digits/dataset/images/classification/forms.py

```diff
@@ -367,3 +368,84 @@ def validate_textfile_test_folder(form, field):
                  "associated with the '(i-1)'th numeric label. (E.g. the string label "
                  "for the numeric label 0 is supposed to be on line 1.)"),
     )
+
+    #
+    # Method - S3
+    #
+
+    s3_endpoint = utils.forms.StringField(
+        u'Training Images',
```

Comment:
It might be clearer if you name this "S3 endpoint URL".

Reply:
Done!